### PR TITLE
Adding r-lmtp and r-schoolmath from CRAN

### PR DESCRIPTION
**Pull request by dillonroach**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-schoolmath/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('schoolmath')"           # [not win]
+    - "\"%R%\" -e \"library('schoolmath')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=schoolmath
+  license: GPL-2.0-only
```
Comment (mfansler) [inline recipes/r-schoolmath/meta.yaml:43]: DESCRIPTION shows `GPL (>= 2)`, so

```suggestion
  license: GPL-2.0-or-later
```

File: recipes/r-schoolmath/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  summary: Contains functions and datasets for math taught in school. A main focus is set to
+    prime-calculation.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```
Comment (mfansler) [inline recipes/r-schoolmath/meta.yaml:48]: Expand

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```

File: recipes/r-lmtp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    - r-progressr
+    - r-schoolmath
+
+test:
+  commands:
+    - $R -e "library('lmtp')"           # [not win]
+    - "\"%R%\" -e \"library('lmtp')\""  # [win]
+
+about:
+  home: https://beyondtheate.com/, https://github.com/nt-williams/lmtp
```
Comment (mfansler) [inline recipes/r-lmtp/meta.yaml:64]: Split URLs

```suggestion
  home: https://beyondtheate.com/
  dev_url: https://github.com/nt-williams/lmtp
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for adding!

Some minor adjusts, then should be good to merge.

File: recipes/r-schoolmath/meta.yaml (hunk 1 above)
Comment (dillonroach) [reply recipes/r-schoolmath/meta.yaml]: Thanks for catching this, updated!

**Comment by mfansler** [comment]
Keep noarch and let win fail (it's fine)

**Review by mfansler — APPROVED** [review]

**Pull request merged**